build-logic/publishing/src/main/kotlin/build-logic.publish-to-tmp-maven-repo.gradle.kts

```diff
@@ -22,6 +22,14 @@ publishing {
             name = "tmp-maven"
             url = uri(localRepoDir)
         }
+        maven {
+            name = "remote"
+            url = uri("https://oss.sonatype.org/service/local/staging/deploy/maven2/")
+            credentials {
+                username = System.getenv("OSSRH_USERNAME")
```

Comment:
Similar to the first comment - How is pgjdbc taking the credentials from the environment?

Reply:
Do not know for sure, but there are additional config files ignored in `.gitignore` which may be providing such details. These are `release.properties`, `*.local.properties`, `pom.xml.*`, `dependency-reduced-pom.xml`.
But then, I do not see these being referred anywhere in the repository either.

Reply:
Let's find this out. Not blocking the PR for this but let's understand this.